src/ImageSharp/Formats/Webp/Lossy/QuantEnc.cs

```diff
@@ -510,51 +527,150 @@ public static void RefineUsingDistortion(Vp8EncIterator it, Vp8SegmentInfo[] seg
         [MethodImpl(InliningOptions.ShortMethod)]
         public static int Quantize2Blocks(Span<short> input, Span<short> output, Vp8Matrix mtx)
```

Comment:
You need to pass `Vp8Matrix` by reference now to avoid stack copy.

Reply:
ah yes, good spot